dirty_cat/tests/test_fuzzy_join.py

```diff
@@ -49,7 +49,7 @@ def test_fuzzy_join(analyzer):
         return_score=True,
         analyzer=analyzer,
     )
-    pd.testing.assert_frame_equal(df_joined2, df_joined3)
+    assert df_joined2.isin(df_joined3).all().all()
```

Comment:
This is because the previous configuration of the how parameter was not aligned with pandas. It is now, so these two are not equal anymore (the column order is different), but they yield the same output.